DQM/L1TMonitor/python/L1TStage2EMTFData_cfi.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+l1tStage2EmtfData = cms.EDAnalyzer(
+    "L1TStage2EMTF",
+    emtfSource = cms.InputTag("emtfStage2Digis"),
```

Comment:
Use different InputTags for the different collections you are going to use in the module to avoid ProductNotFound exceptions